nsfw/nsfw.php

```diff
@@ -80,11 +80,11 @@ function nsfw_addon_settings(&$a,&$s) {
     $s .= '</span>';
     $s .= '<div id="settings_nsfw_expanded" class="settings-block" style="display: none;">';
     $s .= '<span class="fakelink" onclick="openClose(\'settings_nsfw_expanded\'); openClose(\'settings_nsfw_inflated\');">';
-    $s .= '<h3>' . L10n::t('Not Safe For Work (General Purpose Content Filter)') . '</h3>';
+    $s .= '<h3>' . L10n::t('Content Filter(#NSFW tag and more)') . '</h3>';
```

Comment:
Please add a space before the opening parenthesis.

I wouldn't put the hashtag in the name as well, just `Content Filter (NSFW and more)`